src/utils/editableFieldsValidationError.ts

```diff
@@ -0,0 +1,88 @@
+import { AmendmentRecords, UpdatedNextHearingDate, UpdatedOffenceResult } from "../types/Amendments"
+import { isEmptyOrWhiteSpace } from "./isEmptyOrWhitespace"
+import hasNextHearingDateException from "./exceptions/hasNextHearingDateException"
+import { ExceptionCode } from "@moj-bichard7-developers/bichard7-next-core/core/types/ExceptionCode"
+import hasNextHearingLocationException from "./exceptions/hasNextHearingLocationException"
+import { isAsnFormatValid } from "@moj-bichard7-developers/bichard7-next-core/core/phase1/lib/isAsnValid"
+import { DisplayFullCourtCase } from "../types/display/CourtCases"
+import { Exception } from "../types/exceptions"
+import hasAsnException from "./exceptions/hasAsnException"
+
+const nextHearingDateExceptions = [ExceptionCode.HO100102, ExceptionCode.HO100323]
+const nextHearingLocationExceptions = [ExceptionCode.HO100200, ExceptionCode.HO100300, ExceptionCode.HO100322]
+
+///// If there is ASN exception
+// return true when field is empty
+// return true when invalid value is entered
+
+///// If there is no exception
+// return false when field is empty
+// return true when invalid value is entered
+
+// if (amendments.asn) {
+//   return !isAsnFormatValid(amendments?.asn as string) || isEmptyOrWhiteSpace(amendments?.asn as string)
+// }
+const asnValidationError = (exceptions: Exception[], amendments: AmendmentRecords): boolean => {
```

Comment:
I would like if each error would in it's own file

Reply:
Separated into each file